.changeset/unlucky-walls-tie.md

```diff
@@ -1,5 +1,7 @@
 ---
-"@coinbase/onchainkit": minor
+"@coinbase/onchainkit": patch
 ---
 
-**feat**: Swap success state - refetch balances and clear inputs by @0xAlec #1089
+- **patch**: refetch balances and clear inputs after Swap succeeds. By @0xAlec #1089
+- **patch**: Add connect wallet functionality to Swap component for disconnected user. By @abcrane123 #1173
```

Comment:
All changelog have to be in order of merging. so the one that got merged earlier are at the top.

Reply:
changed in latest